spec/spec_helper.rb

```diff
@@ -5,3 +5,14 @@
 
 require 'ruby-osc'
 include OSC
+
+# Support old-style `should` and `stub` monkeypatches (for now)
+RSpec.configure do |config|
+  config.mock_with :rspec do |mocks|
+    mocks.syntax = :should
+  end
+
+  config.expect_with :rspec do |c|
+    c.syntax = :should
+  end
+end
```

Comment:
@nicb Why support old style rspec monkeypatches?

Reply:
@maca  That was to remove the ugly warning during spec running. In fact it took me some time because I never used the old syntax always used the new one. The point is that all specs should be converted to the new syntax and it's a long and boring job. But I'll remove the monkeypatching and put the `expect` syntax on the error raising specs.